Dockerfile

```
@@ -105,51 +123,57 @@ RUN echo -n 00000000-0000-0000-0000-000000000000 > $HOME/.cache/chroma/telemetry
     python -c "import os; from faster_whisper import WhisperModel; WhisperModel(os.environ['WHISPER_MODEL'], device='cpu', compute_type='int8', download_root=os.environ['WHISPER_MODEL_DIR'])"; \
     python -c "import os; import tiktoken; tiktoken.get_encoding(os.environ['TIKTOKEN_ENCODING_NAME'])"; \
+    python -c "import nltk; nltk.download('punkt_tab')"; \
     fi; \
-    chown -R $UID:$GID /app/backend/data/
-
+    fi; \
+    mkdir -p /app/backend/data; chown -R $UID:$GID /app/backend/data/; \
+    rm -rf /var/lib/apt/lists/*;
 
+# Install Ollama if requested
+RUN if [ "$USE_OLLAMA" = "true" ]; then \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)